### PR TITLE
Splits APC attackbys into tool acts, removes some old parts that do nothing and allows crowbars to remove the cell too

### LINKED ISSUE #36740: Broken APCs cannot be deconstructed
If an APC is broken to the point you can examine it and get: `It looks broken and non-functional. It's falling apart! ` (aka it's cover has been removed and it's broken), it cannot be deconstructed via normal means or by applying brute force.

Reproduced today and a few months ago.

```Round ID:  85598 
Server revision compiled on:  2018-03-24 
The following pull requests are currently test merged: 
#36388: 'Shuttles have additional baseturfs, and other minor baseturf changes' by ninjanomnom at commit c64756 
Based off origin/master commit: 385b62 
Revision: 78bfbbdd093d009ad2e76ee917a9c39f0e945f3c 
Server tools version: /tg/station 13 Server v3.2.1.15 
Server tools API version: 3.2.0.2 ```

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
Fixes #36740 ... probably. I couldn't replicate the exact scenario that I've seen in-game after changing a bunch of code around. 

This code is really dire.

:cl: Naksu
code: APC code is slightly less shit.
tweak: APC cells can be removed with a crowbar
/:cl:

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -378,75 +377,133 @@
...
+				QDEL_NULL(integration_cog)
+			return
+		else if (opened!=2) //cover isn't removed
+			opened = 0
+			coverlocked = TRUE //closing cover relocks it
+			update_icon()
+			return
+	else if (!(stat & BROKEN))
+		if(coverlocked && !(stat & MAINT)) // locked...
+			to_chat(user, "<span class='warning'>The cover is locked and cannot be opened!</span>")
```
Comment (YPOQ) [inline code/modules/power/apc.dm]: This should be in crowbar_act

Comment (vuonojenmustaturska) [reply code/modules/power/apc.dm]: fixed

**Comment by vuonojenmustaturska** [comment]
maybe-fix refactors probably don't belong during a freeze, maybe later

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -378,75 +377,142 @@
...
+			"<span class='notice'>You painstakingly start tearing [integration_cog] out of [src]'s guts...</span>")
+			W.play_tool_sound(src)
+			if(W.use_tool(src, user, 100))
+				user.visible_message("<span class='notice'>[user] destroys [integration_cog] in [src]!</span>", \
+				"<span class='notice'>[integration_cog] comes free with a clank and snaps in two as the machinery returns to normal!</span>")
+				playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
+				QDEL_NULL(integration_cog)
+			return
+		else if (opened!=2) //cover isn't removed
+			opened = 0
```
[the remark this answers is not in the document]
Comment (vuonojenmustaturska) [reply code/modules/power/apc.dm]: sure

**Comment by Zeldazackman** [comment]
All this does is introduce a bug wherein an APC's cover cannot be closed.

**Pull request merged**